openff/bespokefit/executor/services/_settings.py

```diff
@@ -62,6 +62,15 @@ class Settings(BaseSettings):
     BEFLOW_OPTIMIZER_WORKER_N_CORES: Union[int, Literal["auto"]] = "auto"
     BEFLOW_OPTIMIZER_WORKER_MAX_MEM: Union[float, Literal["auto"]] = "auto"
     BEFLOW_OPTIMIZER_KEEP_FILES: bool = False
+    """
+    .. deprecated:: 0.2.1
+        use BEFLOW_KEEP_TMP_FILES instead
+
+    Keep the optimizer's temporary files.
+    """
+
+    BEFLOW_KEEP_TMP_FILES: bool = False
```

Comment:
This might be a good time to introduce a name change or do we want to keep it consistent till we break all of these?

Reply:
Oh I see, its a custom thing. Uhm yes I would like machinery like this to handle deprecations, no I don't want to put it in this PR :P